docs/gateway/README.md

```diff
@@ -16,7 +16,10 @@ export MINIO_ACCESS_KEY=azureaccountname
 export MINIO_SECRET_KEY=azureaccountkey
 minio gateway azure
 ```
+## Test using Minio Browser
+Minio Server comes with an embedded web based object browser. Point your web browser to http://127.0.0.1:9000 ensure your server has started successfully.
```

Comment:
Should be `Minio Gateway comes with an embedded browser UI. Point your web browser to http://127.0.0.1:9000 to ensure your server has started successfully. `